src/match_regex.c

```diff
@@ -93,6 +96,12 @@ static void mr_free_match (mr_match_t *m) /* {{{ */
 	mr_free_regex (m->plugin_instance);
 	mr_free_regex (m->type);
 	mr_free_regex (m->type_instance);
+	for (llentry_t *e = llist_head(m->meta); e != NULL; e = e->next)
+	{
+		free (e->key);
```

Comment:
Please search and replace all `free()` with `sfree()`. Ruben just did a tree-wide unification and I'd prefer not to muddy the waters again.